calblink.go

```diff
@@ -1023,4 +1064,6 @@ func main() {
 		fmt.Fprint(dotOut, ".")
 		sleep(time.Duration(userPrefs.pollInterval) * time.Second)
 	}
+	
+}
```

Comment:
Please remove the extra closing bracket.

Reply:
Hi Yes Your are Right I will Remove it